Package.swift

```diff
@@ -0,0 +1,53 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Sentinel",
+    platforms: [
+        .iOS(.v11)
+    ],
+    products: [
+        .library(
+            name: "Sentinel",
+            targets: ["Sentinel"]
+        ),
+        .library(
+            name: "SentinelLoggie",
+            targets: ["SentinelLoggie"]
+        ),
+        .library(
+            name: "SentinelCollar",
+            targets: ["SentinelCollar"]
+        )
```

Comment:
Will this pull Loggie and Collar automatically or will the developer be able to select which tools he wants to install?

Reply:
No it wont pull automatically, the developer will be able to select which tool he wants to include.

Reply:
<img width="696" alt="image" src="https://user-images.githubusercontent.com/13760486/182831817-1d64e177-ae92-4962-b8db-047379fc0eb5.png">

Reply:
great! <3